src/creator/templates/project/{{cookiecutter.name}}/install-spl.ps1

```diff
@@ -1,22 +1,3 @@
-$dependency_json_content = Get-Content -Raw -Path $PSScriptRoot\dependencies.json | ConvertFrom-Json
-if ($dependency_json_content.mandatory.spl_core.version) {
-    $spl_version = $dependency_json_content.mandatory.spl_core.version
-} else {
-    Write-Error "SPL core version not defined in the dependencies.json. Please define 'mandatory.spl-core.version' value."
-    exit 1
-}
-if ($dependency_json_content.mandatory.spl_core.installer) {
-    $spl_installer = $dependency_json_content.mandatory.spl_core.installer
-} else {
-    Write-Error "SPL core installer not defined in the dependencies.json. Please define 'mandatory.spl-core.installer' value."
-    exit 1
-}
-if ($dependency_json_content.mandatory.spl_core.url) {
-    $additional_args = $dependency_json_content.mandatory.spl_core.url
-} else {
-    Write-Warning "SPL core repository url not defined in the dependencies.json. The SPL Github repository will be used by default. Please define 'mandatory.spl-core.url' in case you want to use an SPL mirror repository."
-}
-
-Push-Location $PSScriptRoot
-Invoke-Expression "& { $(Invoke-RestMethod $spl_installer) } $spl_version $additional_args -skipInstall"
+Push-Location "$PSScriptRoot"
+Invoke-Expression "& { $(Invoke-RestMethod https://raw.githubusercontent.com/avengineers/SPL/develop/install.ps1) } v1.6.0 -skipInstall"

```

Comment:
I don't understand why we need this here.

Reply:
you mean general why we need the project creator? In the coookie cutter script I was only adapting to the same behavior we have in SPLDemo.

Reply:
You are right, we do not change the way how to install SPLcore here. This is another issue.